spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2663,16 +2663,16 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
           None
         }
 
-      case HashAggregateExec(
-            _,
-            _,
-            _,
-            groupingExpressions,
-            aggregateExpressions,
-            aggregateAttributes,
-            _,
-            resultExpressions,
-            child) if isCometOperatorEnabled(op.conf, CometConf.OPERATOR_AGGREGATE) =>
+      case aggregate: BaseAggregateExec
+          if (aggregate.isInstanceOf[HashAggregateExec] ||
+            aggregate.isInstanceOf[ObjectHashAggregateExec]) &&
```

Comment:
So for `ObjectHashAggregateExec`, the error message will be something like unsupported aggregate expression?

Reply:
Oh, I saw it in the PR description.